crates/accelerate/src/lib.rs

```diff
@@ -47,28 +45,5 @@ pub fn getenv_use_multiple_threads() -> bool {
     !parallel_context || force_threads
 }
 
-#[pymodule]
-fn _accelerate(m: &Bound<PyModule>) -> PyResult<()> {
-    m.add_wrapped(wrap_pymodule!(nlayout::nlayout))?;
-    m.add_wrapped(wrap_pymodule!(stochastic_swap::stochastic_swap))?;
-    m.add_wrapped(wrap_pymodule!(sabre_swap::sabre_swap))?;
-    m.add_wrapped(wrap_pymodule!(pauli_exp_val::pauli_expval))?;
-    m.add_wrapped(wrap_pymodule!(dense_layout::dense_layout))?;
-    m.add_wrapped(wrap_pymodule!(quantum_circuit::quantum_circuit))?;
-    m.add_wrapped(wrap_pymodule!(error_map::error_map))?;
-    m.add_wrapped(wrap_pymodule!(sparse_pauli_op::sparse_pauli_op))?;
-    m.add_wrapped(wrap_pymodule!(results::results))?;
-    m.add_wrapped(wrap_pymodule!(optimize_1q_gates::optimize_1q_gates))?;
-    m.add_wrapped(wrap_pymodule!(sampled_exp_val::sampled_exp_val))?;
-    m.add_wrapped(wrap_pymodule!(sabre_layout::sabre_layout))?;
-    m.add_wrapped(wrap_pymodule!(vf2_layout::vf2_layout))?;
-    m.add_wrapped(wrap_pymodule!(two_qubit_decompose::two_qubit_decompose))?;
-    m.add_wrapped(wrap_pymodule!(utils::utils))?;
-    m.add_wrapped(wrap_pymodule!(
-        euler_one_qubit_decomposer::euler_one_qubit_decomposer
-    ))?;
-    m.add_wrapped(wrap_pymodule!(
-        convert_2q_block_matrix::convert_2q_block_matrix
-    ))?;
-    Ok(())
-}
+// This involves an import.
+import_exception!(qiskit.exceptions, QiskitError);
```

Comment:
It doesn't actually - I don't know how exactly PyO3 works here (doubtless I'll spend a good hour tomorrow looking it up), but at the time I wrote the comment I was mistaken about this causing an import cycle.

Reply:
Fixed in d64f10127f.